docs/application/native/guides/multimedia/image-edit.md

```diff
@@ -26,7 +26,7 @@ The main features of the Image Util API include:
   You can [decode images](#decode), [encode them](#encode), and [encode animation](#animation) with the following formats:
 
   - Bitmap formats: YUV420, YUV422, RGB888, RGBA8888, BGRA8888, and ARGB8888
-  - Input image formats for decoding: JPEG, PNG, GIF, BMP, and WEBP
+  - Input image formats for decoding: JPEG, PNG, GIF, BMP, WEBP and HEIF
```

Comment:
```suggestion
  - Input image formats for decoding: JPEG, PNG, GIF, BMP, WEBP, and HEIF
```

Reply:
Thank you for your kind comment.
I added missing comma.